recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -255,6 +255,12 @@ do_deploy() {
         echo "dtoverlay=mcp2515-can0,oscillator=${CAN_OSCILLATOR},interrupt=25" >>$CONFIG
     fi
 
+    # PWM support
+    if [ "${ENABLE_PWM}" = "1" ]; then
+        echo "# Enable PWM" >>$CONFIG
+        echo "dtoverlay=pwm,pin=18,func=2" >>$CONFIG
```

Comment:
Might not be a good idea to hardcode this. If I remember this correctly, it conflicts with the i2c audio pins. Also, `18` is the default anyway, so we might just drop it and address a configuration variable later. Same for `func`.

Reply:
That sounds right.